aptos-move/mvhashmap/src/versioned_data.rs

```diff
@@ -361,6 +361,37 @@ impl<K: Hash + Clone + Debug + Eq, V: TransactionWrite> VersionedData<K, V> {
         }));
     }
 
+    /// Versioned write of metadata at a given resource group key (and version).
```

Comment:
nit: docstring can say that it "Returns true if the previously stored metadata has changed" or something like that?